runtime/development/src/lib.rs

```diff
@@ -400,6 +405,27 @@ impl InstanceFilter<Call> for ProxyType {
 				Call::Pools(pallet_pools::Call::submit_solution{..}) |
 				Call::Pools(pallet_pools::Call::execute_epoch{..})
 			),
+			ProxyType::ProxyManagement => matches!(
+				c,
+				Call::Proxy(..)
+			),
+			ProxyType::ProxyKeystoreManagement => matches!(
+				c,
+				Call::ProxyKeystore(pallet_proxy_keystore::Call::add_keys{..}) |
+				Call::ProxyKeystore(pallet_proxy_keystore::Call::revoke_keys{..})
+			),
+			ProxyType::NFTMint => matches!(
```

Comment:
I'm not sure what's the intended use case for this exactly, but it likely should also include `set_metadata` and/or `set_attribute`, since this will often be needed when minting new NFTs?